code/modules/holodeck/holodeck.dm

```diff
@@ -0,0 +1,207 @@
+
```

Comment:
Name this `map_templates.dm` or something else more descriptive

Reply:
now called holodeck_map_templates and updated comment at the top of computer.dm